### PR TITLE
Add easydmarc.com template

### PR DESCRIPTION
**Pull request by avag89**

### CODE REVIEW

File: easydmarc.com.setup.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  "syncRedirectDomain": "easydmarc.com, app.easydmarc.com",
+  "logoUrl": "https://easydmarc.com/img/wordmark.svg",
+  "syncPubKeyDomain": "domainconnect.easydmarc.com",
+  "warnPhishing": true,
+  "description": "Your smart DMARC reporting and monitoring platform. Ensure domain-level security and email deliverability with EasyDMARC's DMARC, SPF, DKIM, and BIMI services",
+  "variableDescription": "Self explanatory in the variable names",
+  "records": [
+    {
+      "groupId": "spf",
+      "type": "TXT",
```
Comment (pawel-kow) [inline easydmarc.com.setup.json:16]: Why not SPFM? This will assure compatibility with other templates or existing SPF

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: easydmarc.com.setup.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
+{
+  "providerId": "easydmarc.com",
+  "providerName": "EasyDMARC",
+  "serviceId": "setup",
+  "serviceName": "EasyDMARC setup",
+  "version": 1,
+  "syncRedirectDomain": "easydmarc.com, app.easydmarc.com",
+  "logoUrl": "https://easydmarc.com/img/wordmark.svg",
+  "syncPubKeyDomain": "domainconnect.easydmarc.com",
+  "warnPhishing": true,
```
Comment (pawel-kow) [inline easydmarc.com.setup.json:10]: The template is signed. Typically no phishing risk shall be present, or?

File: easydmarc.com.setup.json (hunk 1 above)
Comment (avag89) [reply easydmarc.com.setup.json:16]: We need more flexibility for modifying the SPF as EasyDMARC's functional is more complex. EasyDMARC provides [SPF record generator](https://easydmarc.com/tools/spf-record-generator), comprehensive [EasySPF](https://easydmarc.com/tools/easy-spf) feature, and similar functionalities to work with the SPF. With SPFM you can't modify even the policy (that `~all` at the end of the record) or replace it (for example to use `redirect=` instead). `SPFM` is good for email sending sources (vendors) for just putting their SPF's `include` term to make outgoing emails SPF-valid from that source.

File: easydmarc.com.setup.json (hunk 2 above)
Comment (avag89) [reply easydmarc.com.setup.json]: removed

File: easydmarc.com.setup.json (hunk 1 above)
Comment (pawel-kow) [reply easydmarc.com.setup.json:15]: @avag89 understood. These are surely valid reasons. Would be good to add this information (short version) to the description.

The generator does not seem (from my 5s look at it) to take into account existing spf setup. Typically DNS providers, especially hosting providers, are looking after guarantee that the new setup won't nuke the setup needed for their email/newsletter/hosting service. This is how SPFM was born.
Anyway, these are my 2¢, I can merge the template as-is and in the end of the day it's your task to convince DNS providers to onboard it :) Let me know.

Comment (avag89) [reply easydmarc.com.setup.json:15]: Thanks Pawel for clarification. But as you can see we are in this SPF, DKIM, DMARC and BIMI business for a long time (5 years) and we know how to deal with everything related to those :) . So you can freely merge it, the rest is on us.

**Review by pawel-kow — APPROVED** [review]
ok

**Pull request merged**